Formula/gdal2.rb

```diff
@@ -44,107 +41,136 @@ def plugins_subdirectory
     depends_on "cfitsio"
     depends_on "epsilon"
+    depends_on "hdf5"
+    depends_on "jasper"
     depends_on "libdap"
     depends_on "libxml2"
     depends_on "openjpeg"
     depends_on "zstd"
-
+    depends_on "webp"
     # Vector libraries
     depends_on "unixodbc" # OS X version is not complete enough
     depends_on "xerces-c"
-
     # Other libraries
     depends_on "xz" # get liblzma compression algorithm library from XZutils
   end
 
-  if build.with? "swig-java"
-    depends_on :java => ["1.8", :build]
-    depends_on "ant" => :build
-    depends_on "swig" => :build
-  end
-
-  depends_on "sfcgal" => :optional
-
   def configure_args
     args = [
       # Base configuration.
       "--prefix=#{prefix}",
       "--mandir=#{man}",
       "--disable-debug",
+      "--with-libtool",
       "--with-local=#{prefix}",
       "--with-threads",
-      "--with-libtool",
-
-      # GDAL native backends.
-      "--with-pcraster=internal",
-      "--with-pcidsk=internal",
+      # GDAL native backends
       "--with-bsb",
       "--with-grib",
       "--with-pam",
-
-      # Backends supported by OS X.
-      "--with-libiconv-prefix=/usr",
-      "--with-libz=/usr",
-      "--with-png=#{Formula["libpng"].opt_prefix}",
-      "--with-expat=/usr",
-      "--with-curl=/usr/bin/curl-config",
-
-      # Default Homebrew backends.
-      "--with-jpeg=#{HOMEBREW_PREFIX}",
-      "--without-jpeg12", # Needs specially configured JPEG and TIFF libraries.
-      "--with-gif=#{HOMEBREW_PREFIX}",
-      "--with-libtiff=internal",
+      "--with-pcidsk=internal",
+      "--with-pcraster=internal",
+      # Other
+      "--with-libiconv-prefix=#{Formula["libiconv"].opt_prefix}",
+      "--with-libz=#{Formula["libzip"].opt_prefix}",
+      "--with-curl=#{Formula["curl"].opt_bin}/curl-config",
+      "--with-freexl=#{Formula["freexl"].opt_prefix}",
+      "--with-geos=#{Formula["geos"].opt_prefix}/bin/geos-config",
       "--with-geotiff=internal",
-      "--with-sqlite3=#{Formula["sqlite"].opt_prefix}",
-      "--with-freexl=#{HOMEBREW_PREFIX}",
-      "--with-spatialite=#{HOMEBREW_PREFIX}",
-      "--with-geos=#{HOMEBREW_PREFIX}/bin/geos-config",
-      "--with-static-proj4=#{HOMEBREW_PREFIX}",
+      "--with-libtiff=internal",
+      "--with-gif=#{Formula["giflib"].opt_prefix}",
       "--with-libjson-c=#{Formula["json-c"].opt_prefix}",
+      "--with-png=#{Formula["libpng"].opt_prefix}",
+      "--with-spatialite=#{Formula["libspatialite"].opt_prefix}",
+      "--with-expat=#{Formula["expat"].opt_prefix}",
+      "--with-sqlite3=#{Formula["sqlite"].opt_prefix}",
+      "--with-zstd=#{Formula["zstd"].opt_prefix}",
       "--with-xml2=#{Formula["libxml2"].opt_bin}/xml2-config",
-
+      "--with-webp=#{Formula["webp"].opt_prefix}",
+      "--with-netcdf=#{Formula["netcdf"].opt_prefix}",
+      "--with-hdf5=#{Formula["hdf5"].opt_prefix}",
+      "--with-jasper=#{Formula["jasper"].opt_prefix}",
+      "--with-openjpeg=#{Formula["openjpeg"].opt_prefix}",
```

Comment:
If we ship gdal with openjpeg by default maybe we can remove jasper ?

Reply:
@vincentsarago Ok, I will comment on that line.